doc/cabal-commands.rst

```diff
@@ -1010,9 +1010,9 @@ tricky GHC options, etc.).
 
 Run ``cabal check`` in the folder where your ``.cabal`` package file is.
 
-.. option:: -v, --verbose[=n]
+.. option:: -v[n], --verbose[=n]
 
-    Set verbosity level (0–3, default is 1).
+    Control verbosity (n is 0--3, default verbosity level is 1).
```

Comment:
Does our doc engine support things like `--`? I'm afraid that you need to manually insert the `–`.

Reply:
Seems to render better than expected:

https://cabal--9043.org.readthedocs.build/en/9043/cabal-commands.html#cmdoption-v-n

Reply:
Perhaps the `0--3` in the console is the mistake (an artifact)? I've never seen that before verbatim in plain text when `0..3` seems the more usual (or even `0-3`).

I see that `--` is converted to en-dash by [smart quotes of sphinx](https://stackoverflow.com/questions/51075907/display-two-dashes-in-rst-file).